docs/GettingStarted.md

```diff
@@ -160,7 +160,7 @@ and running.
 ### api_test.exe
 This test exercises various eBPF user mode eBPF APIs, including those to load programs,
 enumerate maps and programs etc. This test requires the eBPF user mode service (EbpfSvc), and the
```

Comment:
I think EbpfSvc should also be changed to `EbpfSvc.exe`

Reply:
Here I think it refers to the service name, not the binary file name `ebpfsvc.exe`.
But I can make this change as well if needed.

Reply:
I'm ok with it either way.  (Such a change does not seem to be part of "align formatting" as a subject though so I'm fine with the PR as is.)